yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
Remove this file and run `yarn` from `deprecated-react-native-prop-types`.

```
cd react-native-deprecated-modules
rm yarn.lock
cd deprecated-react-native-prop-types
yarn
```

Reply:
My bad I thought this was a workspace